internal/backuprestore/backup.go

```diff
@@ -565,3 +565,26 @@ func (h *BRHandler) ensureBackupsDeleted(ctx context.Context, backups []velerov1
 	}
 	return true, nil
 }
+
+// CleanupDeleteBackupRequests deletes all DeleteBackupRequest for this cluster from object storage
+// returns: true if all DeleteBackupRequest have been deleted, error
+func (h *BRHandler) CleanupDeleteBackupRequests(ctx context.Context) (bool, error) {
+
+	// List all DeleteBackupRequest CRs created for this cluster
+	deleteBackupRequestList := &velerov1.DeleteBackupRequestList{}
+	if err := h.List(ctx, deleteBackupRequestList); err != nil {
+		return false, fmt.Errorf("failed to list DeleteBackupRequest CRs: %w", err)
```

Comment:
Could we assgin a specific label to the deleteBackupRequest CRs to ensure that we are not deleting other CRs not created by us?
We could use the same label `clusterIDLabel: clusterID` of the backup CR